azure-pipelines.yml

```diff
@@ -0,0 +1,106 @@
+trigger:
+  branches:
+    include:
+    - master
+  tags:
+    include:
+    - "*"
+
+pool:
+  vmImage: 'vs2017-win2016'
+
+variables:
+  solution: '**/*.sln'
+  buildPlatform: 'Any CPU'
+  buildConfiguration: 'Release'
+  isTagBuild: $[startsWith(variables['Build.SourceBranch'], 'refs/tags/')]
+
+jobs:
+- job: 'build'
+  steps:
+  - task: DotNetCoreCLI@2
+    displayName: restore
+    inputs:
+      command: 'restore'
+      feedsToUse: 'select'
+
+  - task: DotNetCoreCLI@2
+    displayName: build solutions
+    inputs:
+      command: 'build'
+  
+  - task: DownloadSEcureFile@1
+    name: devSettings
+    displayName: fetch development settings
+    inputs:
+      secureFile: 'developer-settings.json'
```

Comment:
Resource group name should be constructed from build number (and maybe even branch) to produce unique resource groups for tests.